src/lib/components/multiDropzone/types.ts

```diff
@@ -24,7 +24,7 @@ export const IMAGE_FILES: FileType[] = ['heic', 'bmp', 'jpeg', 'jpg', 'png'];
 export interface UploadedFile {
   id: string;
   name: string;
-  type?: FileType;
+  type?: FileType | string;
```

Comment:
I think you need to fix this issue on the consumer side otherwise you will lose the benefits of defining `FileType`.
This means all applications using the `MultDropZone` need to adapt their types.

Reply:
for example in `app` you would need to change the `type` from string to `FileType` etc.
that way you can ensure that the defined upload file type confirms to the `FileType` in ds.

```
export interface UploadedFile {
  id: string;
  name: string;
  type: string;
  progress: number;
  error?: string;
  token?: string;
  previewUrl?: string;
}
```

Reply:
I agree with @busypeoples, we have to make types stricter on the consumer projects. I think we are already exporting `FileType` from DS, so we can reuse that type on the consumer side.

Reply:
I was trying not to introduce any breaking changes. If we should update on the consumer side, I recommend even removing this type as we are not using it anymore (it was used before to show an icon according to the type of file).

I can update this PR to remove it and fix it on app side. As it's already optional, it is an improvement but we don't need to wait to have this released to fix it.